site/src/index.njk

```diff
@@ -84,39 +84,95 @@ keywords:
       </div>
     </div>
     <hr>
-    <div class='row my-3'>
-      <div class='col'>
-        <div class='field-text field-text-filled' style='width:16rem'>
-          <label class='field-text-icon-leading' for='text'><span
-              class='material-symbols-outlined'>search</span></label>
-          <label class='field-text-label' for='text'>Label text</label>
-          <input class='field-text-input' type='text' id='text' placeholder=' ' pattern="[A-Za-z]{3}">
-          <label class='field-text-icon-trailing' for='text'><span
-              class='material-symbols-outlined material-symbols-outlined-2'>cancel</span></label>
+    <div class='row'>
+      <div class='col'>
+        <span>Filled with supporting text</span>
```

Comment:
Document spans should use proper typography class. Check out the `_typography.scss` for classes. It doesn't currently include all variants, but I'll add them. Best class would probably be `typography-body-large`.